test/Microsoft.AspNet.Mvc.IntegrationTests/MutableObjectModelBinderIntegrationTest.cs

```diff
@@ -26,6 +26,7 @@ public class MutableObjectModelBinderIntegrationTest
 
         private class Order1
```

Comment:
Does this class include any tests that'll hit `IsBindingRequired` violations now?